cfe_internal/update/update_policy.cf

```diff
@@ -249,6 +276,13 @@ bundle agent cfe_internal_update_policy_cpv
       depth_search => u_recurse_basedir("inf"),
       action => u_immediate;
 
+  methods:
+    debian|redhat|amazon_linux|suse|sles::
+      # Only needed on distros with Python-based package modules
+      "remove_possibly_broken_python_symlink" usebundle => cfe_internal_remove_broken_symlink("$(python_symlink)"),
```

Comment:
I think that we don't need to do 2 bundles here, just the one to manage the symlink should be sufficient.